README.md

```diff
@@ -1,17 +1,10 @@
 # MAVLink Router
 
-Route mavlink packets between endpoints.
-
-The usual configuration is to have one connection to the flight stack (either
-on UART or UDP) and other components that can be on UDP or TCP or UART
-endpoints. This is not strictly required and other configurations are possible:
-mavlink-router mainly routes mavlink packets from one endpoint to the other
-endpoints without differentiating what they are. However the user is still able
-to differentiate them by targeting just a subset of the packets by using
-message filters.
-
-TCP endpoints are added automatically if the TCP server is enabled, allowing
-clients to simply connect to mavlink-router without changing its configuration.
+MAVLink Router is an application to distribute [MAVLink](https://mavlink.io/en/)
+messages between multiple endpoints (connections). It actually behaves more
+like a network switch distributing packets to a single port or multiple ports
```

Comment:
I'd say it's more like a network router than a network switch/bridge.  Hence the name :)